# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/spirv-cross-internal/tests/shaders/simple.vert
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/spirv-cross-internal/tests/shaders/simple.vert b/third_party/rust/spirv-cross-internal/tests/shaders/simple.vert
deleted file mode 100644
--- a/third_party/rust/spirv-cross-internal/tests/shaders/simple.vert
+++ /dev/null
@@ -1,17 +0,0 @@
-#version 310 es
-
-layout(std140) uniform uniform_buffer_object
-{
-    mat4 u_model_view_projection;
-    float u_scale;
-};
-
-layout(location = 0) in vec4 a_position;
-layout(location = 1) in vec3 a_normal;
-layout(location = 0) out vec3 v_normal;
-
-void main()
-{
-    v_normal = a_normal;
-    gl_Position = u_model_view_projection * a_position * u_scale;
-}